js/example.js

```diff
@@ -13,6 +13,10 @@ module.exports = function (styleguide, filePath) {
   const buildPath = styleguide.path.build()
   let data = resolver.data(buildPath, filePath)
 
+  if (Object.keys(data).length === 0) {
```

Comment:
In that case, I think it'd be better to throw an error here with the more meaningful message to be captured by try/catch. That way, additional messaging can be centralized in a single place (e.g. appending ` at [${filePath}]!`).